tools/fast/environment_devnet.go

```diff
@@ -0,0 +1,182 @@
+package fast
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"sync"
+
+	cid "github.com/ipfs/go-cid"
+	logging "github.com/ipfs/go-log"
+	iptb "github.com/ipfs/iptb/testbed"
+
+	"github.com/filecoin-project/go-filecoin/address"
+)
+
+// EnvironmentDevnet is a FAST lib environment that is meant to be used
+// when working with kittyhawk devnets.
```

Comment:
Since this code is public and fairly generic, it would be better to comment about specifically what features of an infra this relies on.

More generally, this file would benefit from a top-level comment outlining its intended functionality/benefit, its environmental expectations, access needs etc.